ansible/roles/ansible-datamover-api/tasks/install_rhel.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: Remove old trilio repo
```

Comment:
@trilio-shiwanand 
Why we are needing to remove old file and add new file?
Can't we handle this repo file change through single task?

Reply:
@shyam-biradar if more than 1 repo files are there DevOps script uses the older one to fetch and download the packages.

Can't we handle this repo file change through the single task?
---> In 5.0 we have followed the naming conventions triliovault instead of trilio, hence handling the same task is not preferred.

Reply:
No need to follow this naming convention for repo name. Please use same repo name.
You can add your changes accrodingly.